viewing_party/party.py

```diff
@@ -1,23 +1,183 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+
+    new_movie = {}
+
+    # check that all necessary movie characteristics exist before creating new movie
+    if title and genre and rating:
+        new_movie["title"] = title
+        new_movie["genre"] = genre
+        new_movie["rating"] = rating
+    else:
+        new_movie = None
+
+    return new_movie
+
+def add_to_watched(user_data, movie):
+
+    # add movie to watched list
+    user_data["watched"].append(movie)
+
+    return user_data

```

Comment:
Finding a balance of whitespace inside of a function can be hard, and is often subjective. That said, for a very short function like this with a single statement and return, I'd recommend removing the new lines on line 18 & 21 since this function is so small that everything is tightly related. This applies to the add_to_watchlist function below as well.

Reply:
Thank you for this note! Yes, I'm definitely still finding my footing re: whitespace and readability, so this note is well-received.